### DISCUSSION TITLE
BlueNova - BN13V-108-1.4k BT

### DISCUSSION THREAD

**Discussion by JohnnyVe** [Louisvdw/dbus-serialbattery#446] [Q&A]
Hi,

Would the BlueNova BN13V-108-1.4k BT be a candidate for this project?

In this video https://www.youtube.com/watch?v=QtkmWLSUqRo&t=435s at about 6:00, mention is made of the serial RS-485 port and how it is possible to read details from the BN13V-108-1.4k BT battery to a remote monitoring system.

Thanks
johnny

**Comment by Louisvdw** [comment]
Hi @JohnnyVe 
From what they mention there it would be possible yes. But the [specs](https://bluenova.co.za/wp-content/uploads/2019/03/BN13V-108-1.4k.pdf) don't mention the RS485 port or what protocol it use for communication.

**Comment by JohnnyVe** [comment]
Hi @Louisvdw,
Thanks for the feedback (and the great project!)

I have opened a support call with BlueNova, but ja, not holding my breath. :-)

**Comment by JohnnyVe** [comment]
This is the feedback I received from BlueNova:

Unfortunately this port and its protocol has not been used by anyone, especially Victron,

since the serial comms with Victron is CAN and not RS485. Personally I would invest rather in 

a Victron Smart BMV unit, or a Victron Smart Shunt. This will allow you to add more 13V batteries 

at a later stage without the need for the worries of serial communication between batteries.

**Comment by JohnnyVe** [comment]
After seeking some more details from BlueNova on the port and protocols in use, they did provide some further feedback:

The RS485 RJ11 ports on the battery needs to be used in conjunction with a USB to RS485 converter as well as a BMS monitoring tool. This cannot integrate directly with Victron as far as I understand due to the fact that a communications port needs to be opened by a host like a computer. I can provide the RJ11 pin outs and a possible model of tool that will work and also recommend a program, but this is not something that will be self controlling for integration into Victron. The Victron Cerbo GX does have RS485 inputs via a dongle but this is for sensors and not BMS connection.
 
Pin outs: 1. Through the USB turn to RS485 communication line to connect 485 communication port, use the RJ11 network cable to connect, pay attention to divide AB line, RJ11 pins are defined to A=1 B=6.

![image](https://user-images.githubusercontent.com/124240779/218109185-05131a3f-1f6b-4e78-b5cf-4dede1c36bf6.png)


USB to RS485 converter:
![image](https://user-images.githubusercontent.com/124240779/218109313-effec9ec-916c-49d8-9270-3d68a02d5463.png)


Possible BMS tool:
![image](https://user-images.githubusercontent.com/124240779/218109336-66410649-9da5-42fc-93a3-d284204eea1e.png)

**Comment by JohnnyVe** [comment]
I have now asked them to elaborate on the "WM BMS" or is this something that is a known product in the market?

**Comment by mr-manuel** [comment]
You will need the RS485 protocol description to be able to understand how the data should be received and which data belongs to which value.
